FlashLFQ/FlashLFQ.csproj

```diff
@@ -1,14 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Would you move <PlatformTarget>x64</PlatformTarget>  to the net45 propertyGroup below?